runtime/gc_base/GCExtensions.cpp

```diff
@@ -309,7 +309,7 @@ MM_GCExtensions::releaseNativesForContinuationObject(MM_EnvironmentBase* env, j9
 			Assert_GC_true_with_message2(env, (NULL == continuation), "Continuation expected to be NULL, but it is %p, from Continuation object %p\n", continuation, objectPtr);
 		}
 	} else {
-		getJavaVM()->internalVMFunctions->freeContinuation(vmThread, objectPtr);
+		getJavaVM()->internalVMFunctions->freeContinuation(vmThread, objectPtr, true);
```

Comment:
`true` is not an appropriate argument for a parameter of type `BOOLEAN`: it should be `TRUE`.

Reply:
Opened https://github.com/eclipse-openj9/openj9/pull/17398 to address this concern.